plugins/github-pull-requests-board/src/components/EntityTeamPullRequestsContent/EntityTeamPullRequestsContent.tsx

```diff
@@ -24,6 +24,7 @@ import { PullRequestCard } from '../PullRequestCard';
 import { usePullRequestsByTeam } from '../../hooks/usePullRequestsByTeam';
 import { PRCardFormating } from '../../utils/types';
 import { DraftPrIcon } from '../icons/DraftPr';
+import { ArchivedRepoIcon } from '../icons/ArchivedRepo';
```

Comment:
the build error is because there isn't an `index.ts` exporting the component (`../icons/ArchivedRepo` is a folder, not a ts file). You could also just add `/ArchivedRepo` (duplicating it so it points to the file) to the end of the string on this row to get the same result

Reply:
Maybe you guessed, but I just copy pasted things. Will look into using one of the material-ui icons instead